README.md

```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+## hw01
+
+## Regina Catipon Bio
+
+
+<div><img src="https://rkcatipon.files.wordpress.com/2015/02/10014802_10153266423432857_4524792863877171598_o.jpg" width="200" height="300" align="left"></div>
+
+Hi, I’m **Regina**. I’m one part verve and two parts curiosity.
+
+A former journalist turned social scientist, I am now  pursuing a masters in Computational Social Science from the University of Chicago to explore information dissemination online.
+
+### Work
+Before academia, I worked as the Product Marketer for the data analytics platform <a href="https://www.scraawl.com/">Scraawl</a> and collaborated on research projects for DARPA as a social scientist, researching the impact of fake news and bots.
```

Comment:
Like here, `[Scraawl](https://www.scraawl.com/)` produces the same output. [Scraawl](https://www.scraawl.com/)

Markdown syntax is preferable since it can automatically work regardless of the output document format. You can use R Markdown to generate Markdown documents, HTML, PDF, Word, RTF, etc. But if you used HTML syntax in the RMarkdown document and rendered to PDF, it would not appear correctly.